tests/runtime/engine/runner.py

```diff
@@ -445,8 +445,16 @@ def print_befores_and_after_output():
                 print('\tExpected REGEX: ' + test.expect)
                 print('\tFound:\n' + to_utf8(output))
             elif test.expect_mode == "json":
-                print('\tExpected JSON:\n' + json.dumps(json.loads(open(test.expect).read()), indent=2))
-                print('\tFound:\n' + json.dumps(json.loads(output), indent=2))
+                try:
+                    expected = json.dumps(json.loads(open(test.expect).read()), indent=2)
+                except json.decoder.JSONDecodeError as err:
+                    expected = "Could not parse JSON: " + str(err)
```

Comment:
A bit odd that checked-in json is invalid, but I guess it could help for development